sdk/keyvault/keyvault-certificates/swagger/README.md

```diff
@@ -7,17 +7,19 @@ typescript:
   package-name: "@azure/keyvault-certificates"
 disable-async-iterators: true
 api-version-parameter: choice
-use-core-v2: false
+use-core-v2: true
 v3: true
 use-extension:
-  "@autorest/typescript": "6.0.0-beta.15"
+  "@autorest/typescript": "6.0.0-beta.19"
 azure-arm: false
 generate-metadata: false
 add-credentials: false
+core-http-compat-mode: true
 license-header: MICROSOFT_MIT_NO_VERSION
 input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e2ef44b87405b412403ccb005bfb3975411adf60/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/certificates.json
 output-folder: ../
 source-code-folder-path: ./src/generated
 hide-clients: true
 package-version: 4.5.0-beta.1
+openapi-type: data-plane
```

Comment:
I am curious, what does this change mean?

Reply:
There was an unexpected behavior in the codegen side, so by adding this flag that ensures that this package is treated as a data-plane one. Even if that behavior is now fixed, keeping this flag shouldn't be an issue and will make the openapi-type more explicit.